scriptmodules/ports/alephone.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+# 
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+# 
+# See the LICENSE.md file at the top-level directory of this distribution and 
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="alephone"
+rp_module_desc="AlephOne - Marathon Engine"
+rp_module_menus="4+"
+rp_module_flags="!x11 !mali"
+
+function depends_alephone() {
+    getDepends libboost-all-dev libsdl1.2-dev libsdl-net1.2-dev libsdl-image1.2-dev libsdl-ttf2.0-dev libspeexdsp-dev libzzip-dev libavcodec-dev libavformat-dev libavutil-dev libswscale-dev
+}
+
+function sources_alephone() {
+    wget -O- -q https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20150620/AlephOne-20150620.tar.bz2 | tar -xvj --strip-components=1
+}
+
+function build_alephone() {
+    ./configure --prefix="$md_inst" --with-boost-libdir=/usr/lib/arm-linux-gnueabihf
+    make clean
+    make
+}
+
+function install_alephone() {
+    make install
+}
+
+function configure_alephone() {
+    wget https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20150620/Marathon-20150620-Data.zip
+    wget https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20150620/Marathon2-20150620-Data.zip
+    wget https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20150620/MarathonInfinity-20150620-Data.zip
+    unzip Marathon-20150620-Data.zip
+    unzip Marathon2-20150620-Data.zip
+    unzip MarathonInfinity-20150620-Data.zip
+    mkRomDir "ports"
+    mkRomDir "ports/$md_id/"
```

Comment:
Please put the mkrom folders first and only install the game data if it isn't already installed. You also should clean up any zip files after. configure could be run more than once, so if the existing wget downloads are there it would save to zip.1 files currently.  see lr-tyrquake (it doesn't need to be in a separate function that is just done in lr-tyrquake so it can be used shared by the standalone tyrquake port)